proto/destination.proto

```diff
@@ -69,6 +70,9 @@ message Update {
     // `no_endpoints(exists: true)` indicates that the service does exist and
     // the client MUST NOT fall back to an alternate service discovery method.
     NoEndpoints no_endpoints = 3;
+
+    // Set the service profile, replacing the previous profile if there was one.
+    profile.ServiceProfile profile = 4;
```

Comment:
As discussed, I think we probably ought to bite the bullet and split out a separate Service for profile updates